docs/products/kafka/howto/enable-governance.md

```diff
@@ -0,0 +1,107 @@
+sidebar_label: Enable governance
+limited: true
+---
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Enable governance in Aiven for Apache Kafka® to establish a secure and compliant framework to manage your Aiven for Apache Kafka clusters efficiently.
+
+## Prerequisites
+
+- This is a [limited availability feature](/docs/platform/concepts/beta_services). To try
+  it out, contact the sales team at [sales@aiven.io](mailto:sales@aiven.io).
+- You need [super admin](/docs/platform/howto/make-super-admin) permissions to
+  enable governance.
+
+## Enable governance
+
+1. Access the [Aiven Console](https://console.aiven.io/) and click **Admin**.
+1. On the organization page, click <ConsoleLabel name="Apache Kafka governance"/>.
+1. Click **Enable governance**.
+1. Select a user group to manage topics. This group is the default group and
+   owns all topics in the organization.
+   - If no user group exists, click **Create new user group**.
+     - Enter the group name and description.
+     - Click **Create group**.
+   - Select the created user group.
+1. Click **Next**.
+1. Define resource visibility for the Apache Kafka topic catalog.
+1. Select **Project** or **Organization** visibility.
+
+   - **Project**: Users can view Aiven for Apache Kafka resources in the Apache Kafka
+     topic catalog if they are project members.
+   - **Organization**: Users can view all Aiven for Apache Kafka resources across the
+     organization in the Apache Kafka topic catalog, regardless of their access to
+     individual projects.
+
+1. Click **Next**.
+1. Set global default topic configurations:
+   - To use the default settings, click **Keep defaults** and review the default
+     values on the confirmation window.
+   - Click **Enable governance**.
+
+   Alternatively, to customize:
+
+   - Enter your global default topic configurations.
+   - Click **Enable governance** to apply your changes.
+
+    :::note
+    Global topic configurations apply only to new topics created after the policies are
+    updated. Existing topics are not be affected by these changes.
+    :::
+
+### Change the default group
+
+To change the default user group after enabling governance:
+
+1. On the organization page, click <ConsoleLabel name="Apache Kafka governance"/>.
+1. Click **Change** next to **Default user group**.
+1. Select a new user group from the list.
+1. If no suitable group exists, [create a group](/docs/platform/howto/manage-groups#create-a-group)
+   and select it.
+1. Click **Save**.
+
+### Update global topic configurations
+
+To change global topic configurations after enabling governance:
+
+1. On the organization page, click <ConsoleLabel name="Apache Kafka governance"/>.
+1. Click **Change** next to **Global Apache Kafka topic policies**.
+1. Update the global default topic configurations as needed.
+   - Modify retention policies, partition strategies, or other settings.
+1. Click **Save**.
+
+### Impact of enabling governance
```

Comment:
How about including this section at the beginning of the article, before prerequisites?

Reply:
I did think about that, but I wasn't too sure. I will see how users are using this feature and then, based on feedback, will move.

Reply:
@harshini-rangaswamy  I agree with Dorota. Either we tell this before the enable section, or as the intro of the task, but bringing later is not user friendly. 

It's similar to deletion tasks saying "This action cannot be reversed" after the last step, for instance, which a classic.

Reply:
Thanks! Moved this to the top.